sdks/python/src/opik/evaluation/models/litellm_chat_model.py

```diff
@@ -0,0 +1,68 @@
+import logging
+from typing import Any, Dict, List
+
+import litellm
+from litellm.types.utils import ModelResponse
+
+from . import base_model
+
+LOGGER = logging.getLogger(__name__)
+
+
+class LiteLLLMChatModel(base_model.OpikBaseModel):
+    def __init__(
+        self,
+        model_name: str = "gpt-3.5-turbo",
+        **kwargs: Any,
```

Comment:
**kwargs here is not very intuitive, to check if `top_logprobs` are supported we need to call something like
```python
model = LiteLLMChatModel(model_name="...", top_logprobs=10)
```
even if later we might need to use another value or not use this value in some calls at all.

What I think will be a bit better here is:
1. Rename `**kwargs` to `**completion_kwargs` for arguments we want to automatically add to completion calls.
2. Add an argument `must_support_arguments: Optional[List[str]] = None`. On our end, we'll check that the engine supports both the keys from `**completion_kwargs` and values from `must_support_arguments`. If something is not supported, we'll raise an error from `__init__` function (I mentioned error raising in another comment).